### PR TITLE
Addition to the original TMD Siren

### PR DESCRIPTION
**Pull request by Officer-Pringle**
Moved my siren wav into the original sound dir. and added the phaser tone to the original TMD Siren File as requested

### CODE REVIEW

**Comment by SuperMighty1** [comment]
Missing a comma at the end of the "PHSR" tone

File: lua/autorun/photon/library/emv_sirens.lua [hunk 1]
```diff
@@ -36,6 +36,7 @@ local sirenTable = {
 		Set = {
 			{Name = "WAIL", Sound = "emv/sirens/federal sig tm/emv_wail.wav", Icon="wail"},
 			{Name = "YELP", Sound = "emv/sirens/federal sig tm/emv_yelp.wav", Icon="yelp"},
+			{Name = "PHSR", Sound = "emv/sirens/federal sig tm/pemv_tmjingle.wav", Icon = "phaser"} -- edit by pringle
```
Comment (SuperMighty1) [inline lua/autorun/photon/library/emv_sirens.lua]: Missing a comma at the end of the line.

**Review by SuperMighty1 — CHANGES_REQUESTED** [review]
Missing a comma at the end of the "PHSR" line.

**Review by SuperMighty1 — APPROVED** [review]
tasty

**Comment by Officer-Pringle** [comment]
Cant believe I did such a noob thing

**Pull request merged**